src/Stratis.Bitcoin.Features.SignalR/Events/StakingInfoClientEvent.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Stratis.Bitcoin.EventBus;
+using Stratis.Bitcoin.Features.Miner.Models;
+
+namespace Stratis.Bitcoin.Features.SignalR.Events
+{
+    /// <summary>
+    /// Marker type for Client
+    /// </summary>
+    public class StakingInfo
+    {
+    }
+
+    public class StakingInfoClientEvent : GetStakingInfoModel, IClientEvent
+    {
+        public StakingInfoClientEvent()
```

Comment:
Is this needed? Perhaps for serialization?

Reply:
Done